src/components/todo/TodoItem.tsx

```diff
@@ -0,0 +1,99 @@
+import todoApi from 'apis/todo';
+import { TodosContext } from 'contexts/todosContext';
+import type { ChangeEventHandler, FC, MouseEventHandler } from 'react';
+import { useContext, useState } from 'react';
+import { Todo } from 'types/Todo';
+
+const TodoItem: FC<Todo> = ({ id, isCompleted = false, todo }) => {
+  const { setTodos } = useContext(TodosContext);
+  const [isChecked, setIsChecked] = useState(isCompleted);
+  const [inputValue, setInputValue] = useState(todo);
+  const [editMode, setEditMode] = useState(false);
```

Comment:
isChecked는 boolean인게 바로 와닿아서 좋았습니다~
editMode도 동일하게 boolean 값을 가지니 비슷하게 맞춰주면 어떨까요?

모든 boolean 값을 가진 변수는 is나 has로 시작하도록 일관성있게 사용하면 더 좋을 것 같습니다! 

```suggestion
  const [isChecked, setIsChecked] = useState(isCompleted);
  const [inputValue, setInputValue] = useState(todo);
  const [isEditMode, setIsEditMode] = useState(false);
```